src/synthetix/StakingRewards.sol

```diff
@@ -162,11 +162,14 @@ contract StakingRewards is IStakingRewards, Pausable, ReentrancyGuard {
         emit Recovered(tokenAddress, tokenAmount);
     }
 
-    function setRewardsDuration(uint256 _rewardsDuration) external onlyOwner {
-        require(
-            block.timestamp > periodFinish,
-            "Previous rewards period must be complete before changing the duration for the new period"
-        );
+    function setRewardsDuration(uint256 _rewardsDuration) external onlyOwner updateReward(address(0)) {
```

Comment:
Need to update the README as well, as it lists all the changes from the Synthetix version.